browser/react/components/Login.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { login, logout } from '../../redux/reducers/auth';
+
+/* ----------------- COMPONENT ------------------ */
+
+class Login extends React.Component {
+  constructor (props) {
+    super(props);
+    this.onLoginSubmit = this.onLoginSubmit.bind(this);
+  }
+
+  // Set the background style & size for just this component
+  componentDidMount () {
+    document.body.style.background = 'url(/images/background.png) no-repeat top center fixed';
+    document.body.style.backgroundSize = 'cover';
+  }
+
```

Comment:
couldn't the background image just be for a div that's 100% height and width... give the div an id and only render it on the login page